bety/init-bety.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+
```

Comment:
would it make sense to allow for different entries in here:
nothing - start bety
update - pull latest from remote hosts
init - initialize database